java/src/main/java/com/kevinthorley/csi280/project01/ClassRepo.java

```diff
@@ -15,11 +15,18 @@ public ClassRepo() {
   }
 
   public Section findById(String id) {
-    // TODO: find a section by its id
-    return null;
-  }
+      //  find a section by its id COMPLETED
+ 	for (int i = 0; i < sections.size(); i++) {
+ 	if (id == sections.get(i).getId())
+ 	return (sections.get(i));
+ 	}
+      return null;
+    }
 
   public void addSection(Section section) {
-    // TODO: add a section to the repository
+    // TODO: add a section to the repository COMPLETED
+	this.sections.add(section);
+    return null;
   }
-}
```

Comment:
I had to re-add this closing brace to get it to compile. Assuming this was a typo?